iam-proxy-italia-project/backends/cieoidc/endpoints/authorization_endpoint.py

```diff
@@ -0,0 +1,186 @@
+        if not value:
+            raise ValueError(f"{label} is empty in {self.__class__.__name__}")
+        return value
+
+    def _validate_configs(self):
+        """
+        Validates essential configuration fields for the authorization endpoint.
+        """
+        self._require_config_field(
+            ["endpoints", "authorization_endpoint"], "Authorization endpoint")
+        self._require_config_field(
+            ["endpoints", "authorization_endpoint", "config"], "Authorization endpoint config")
+        self._require_config_field(
+            ["endpoints", "authorization_endpoint", "config", "metadata"], "Metadata")
+        self._require_config_field(
+            ["endpoints", "authorization_endpoint", "config", "metadata", "openid_relying_party"],"OpenId Relying Party")
+        self._require_config_field(
+            ["endpoints", "authorization_endpoint", "config", "metadata", "openid_relying_party","client_id"],"Client ID")
+        self._require_config_field(
+            ["endpoints", "authorization_endpoint", "config", "metadata", "openid_relying_party", "redirect_uris"],
+            "Redirect URI")
+
+
+    def endpoint(self, context, *args):
+        """
+        Handles the authentication response from the OP.
+        :type context: satosa.context.Context
+        :type args: Any
+        :rtype: satosa.response.Response
+
+        :param context: SATOSA context
+        :param args: None
+        :return:
+        """
+        logger.debug(
+            f"Entering method: {inspect.getframeinfo(inspect.currentframe()).function}. Params [context {context}]"
+        )
+
+        authz_endpoint = "http://cie-provider.org:8002/oidc/op/authorization" # TODO Insert this property into config file?
+
+        _timestamp_now = int(datetime.now().timestamp())
+
+        try:
+            authz_data = dict(
+                iss=self.config["metadata"]["openid_relying_party"]["client_id"],
+                scope="openid", # TODO Insert this property into config file?
+                redirect_uri=self.config["metadata"]["openid_relying_party"]["redirect_uris"][0],
+                response_type="code",# TODO Insert this property into config file?
+                nonce=random_string(32),
+                state=random_string(32),
+                client_id=self.config["metadata"]["openid_relying_party"]["client_id"],
+                endpoint="http://cie-provider.org:8002/oidc/op/authorization", # TODO Insert this property into config file?
+                acr_values="https://www.spid.gov.it/SpidL2", # TODO Ask this to Giuseppe because into Django this variable is empty or not? OIDCFED_ACR_PROFILES = getattr(settings,"OIDCFED_ACR_PROFILES",AcrValues.l2.value)
+                iat=_timestamp_now,
+                exp=_timestamp_now + 60, # TODO Ask this to Giuseppe because into Django this variable is empty or not? getattr(settings, "RP_REQUEST_EXP", 60)
```

Comment:
is those 0 about millisecons or seconds? I suppose milliseconds, please check, validate and patch if needed